examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -35,6 +39,18 @@ CHIP_ERROR PairingCommand::Run()
     GetExecContext()->commissioner->RegisterDeviceAddressUpdateDelegate(this);
     GetExecContext()->commissioner->RegisterPairingDelegate(this);
 
+#if CONFIG_PAIR_WITH_RANDOM_ID
+    // Generate a random remote id so we don't end up reusing the same node id
```

Comment:
Do we need any checks that we do not have random colision (to at least error out)? 
Or is this test-only and ok to assume everything ok?

strangely enough, the docs for random() on my linux say only "32 bit" even though I am on a 64 bit platform and return type is long.

Reply:
Collision with what, exactly?

And this is not exactly "test-only" in the sense that we turn CONFIG_PAIR_WITH_RANDOM_ID _off_ in CI.  But the idea here is that people using chip-tool for test events don't end up with multiple nodes all with the same node id...

> strangely enough, the docs for random() on my linux say only "32 bit" even though I am on a 64 bit platform and return type is long.

Same thing on Mac.  I considered generating two random numbers and building a NodeId out of them, but decided it wasn't worth it for the thing we are trying to address here.